project-base/storefront/components/Pages/Order/ContactInformation/ContactInformationFormContent.tsx

```diff
@@ -60,6 +62,17 @@ export const ContactInformationFormContent: FC = () => {
                         onChange: (event) => updateContactInformation({ note: event.currentTarget.value }),
                     }}
                 />
+
+                <CheckboxControlled
```

Comment:
This checkbox will be rendered every time. Even when the shop is in a language that does not support Heureka, or the Heureka itself is not activated. 

For me it's far from ideal, but I would like @pk16011990 to make a final decision.

Reply:
that is a good point. The condition was not hard to implement so I have added it in 252bf08138 :innocent: